.github/workflows/workflow-complete.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       uses: envoyproxy/toolshed/gh-actions/status@actions-v0.0.7
       with:
         authToken: ${{ secrets.GITHUB_TOKEN }}
-        context: ${{ github.event.workflow.name }}
+        context: Verify/examples
```

Comment:
Why is `Verify/examples` hardcoded here? shouldn't it be fetched from the github workflow name?

Reply:
that was ~the existing problem

the workflow name is now `Publish & verify` which is unhelpful when you want both publish and verify jobs with separate status

this is a temp workaround to restore previous behaviour until i figure something better for adding additonal jobs